coverage-tests.js

```diff
@@ -55,6 +55,22 @@ test('check window', {
     'with css stitching': {config: {stitchMode: 'CSS', baselineName: 'TestCheckWindow'}},
     'with scroll stitching': {config: {stitchMode: 'Scroll', baselineName: 'TestCheckWindow_Scroll'}},
     'with vg': {vg: true, config: {baselineName: 'TestCheckWindow_VG'}},
+    'web android': {
+      env: {
+        device: 'Pixel 3a XL',
+        browser: 'chrome',
+      },
+      page: 'HelloWorld',
+      config: {baselineName: 'check window on mobile web android'}
```

Comment:
no reason to specify the baseline name unless you really want to, because it will be generated out of the test name. Also to generate a better test name you would call variants `on web android` instead of `web android`, the same way as we call vg variant `with vg` instead of `vg`.
It is definitely not a deal-breaker, just an idea behind it.